consensus/vote.go

```diff
@@ -12,6 +13,13 @@ import (
 func (consensus *Consensus) receiveVote(neighborID string, height uint32, blockHash common.Uint256) error {
 	log.Debugf("Receive vote %s for height %d from neighbor %v", blockHash.ToHexString(), height, neighborID)
 
+	if consensus.localNode.GetSyncState() == pb.PERSIST_FINISHED {
+		expectedHeight := consensus.GetExpectedHeight()
+		if math.Abs(float64(height)-float64(expectedHeight)) > acceptVoteHeightRange {
```

Comment:
A little suggestion: `math.Abs(float64(height-expectedHeight))`

Reply:
It will get wrong value if height < expectedHeight